core/backend/eslint.config.js

```diff
@@ -0,0 +1,20 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
```

Comment:
does the lint script in the package.json need to specify the file glob as an input

Reply:
The glob in the config file specifies what rules apply to what files. The glob in the lint script says which files to lint

Reply:
Yes, was struggling to phrase that in my head. Thanks Jake